digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -777,94 +675,66 @@ class RoadAddressService(roadLinkService: RoadLinkService, roadwayDAO: RoadwayDA
 
   def filterRelevantChanges(roadAddresses: Seq[RoadAddress], allChanges: Seq[ChangeInfo]): Seq[ChangeInfo] = {
     throw new NotImplementedError("Will be implemented at VIITE-1569, and probably this method should be moved to VVHChangeInfoClient")
-//    val groupedAddresses = roadAddresses.groupBy(_.linkId)
-//    val timestamps = groupedAddresses.mapValues(_.map(_.adjustedTimestamp).min)
-//    allChanges.filter(ci => timestamps.get(ci.oldId.getOrElse(ci.newId.get)).nonEmpty && ci.vvhTimeStamp >= timestamps.getOrElse(ci.oldId.getOrElse(ci.newId.get), 0L))
+    //    val groupedAddresses = roadAddresses.groupBy(_.linkId)
+    //    val timestamps = groupedAddresses.mapValues(_.map(_.adjustedTimestamp).min)
+    //    allChanges.filter(ci => timestamps.get(ci.oldId.getOrElse(ci.newId.get)).nonEmpty && ci.vvhTimeStamp >= timestamps.getOrElse(ci.oldId.getOrElse(ci.newId.get), 0L))
   }
 
   def applyChanges(roadLinks: Seq[RoadLink], allChanges: Seq[ChangeInfo], roadAddresses: Seq[RoadAddress]): Seq[LinkRoadAddressHistory] = {
     throw new NotImplementedError("Will be implemented at VIITE-1569")
```

Comment:
VIITE-1569 has been resolved already at 18.09.2018. Maybe this function should be removed.